index.html

```diff
@@ -6003,6 +5861,113 @@ <h3>RemoteDocument</h3>
     </section>
   </section> <!-- end of Remote Document and Context Retrieval -->
 
+  <section class="changed">
+    <h2>HTML Content Algorithms</h2>
+    <p class="note">This section describes features available
```

Comment:
```suggestion
    <p class="note">This section describes optional features available
```
This is redundant with the "MAY" below, but just to be on the safe side?